cmd/juju/action/run.go

```diff
@@ -119,7 +119,7 @@ The value for the "time" param will be the string literal "1000".
 `
 
 // ActionNameRule describes the format an action name must match to be valid.
-var ActionNameRule = regexp.MustCompile("^[a-z](?:[a-z-]*[a-z])?$")
+var ActionNameRule = regexp.MustCompile("^[a-z0-9](?:[a-z0-9-]*[a-z0-9])?$")
```

Comment:
I wonder if there is a reason for why we define it here again? We need to have one point of truth and we have already defined the same thing in charms.v6, non?
FWIW most of our entities have their naming rules defined in names package. Instinctively, this is where I would have looked for action name rules...

I would have expected that we have  unit tests for names validity close to the definition... i.e. in one place and not duplicated across all libraries..